ops/provision/modules/k8s/main.tf

```diff
@@ -0,0 +1,90 @@
+terraform {
+  required_providers {
+    rancher2 = {
+      source = "rancher/rancher2"
+      version = "1.11.0"
+    }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
+    }
+  }
+}
+
+
+provider "helm" {
+  kubernetes {
+    config_path = var.kubeconfig
+  }
+}
+
+provider "kubectl" {
+  config_path = var.kubeconfig
+}
+
+provider "kubernetes" {
+  config_path = var.kubeconfig
+}
+
```

Comment:
👇 I don't see this resource, nor a `ingress-nginx` namespace. Is this yet to be applied, or no longer used, or am I misunderstanding something?

Reply:
its deployed. I see it 
<img width="1728" alt="Screen Shot 2022-10-24 at 14 22 11" src="https://user-images.githubusercontent.com/1054448/197632105-4fef8bc8-415f-4d42-b858-81e0d3b984f1.png">

Reply:
👍 i guess i had a too-restrictive filter selected